src/main/webapp/view/video/video.js

```diff
@@ -45,7 +43,7 @@ function onYouTubeIframeAPIReady() {
 /** `event` plays the YouTube video. */
 function onPlayerReady(event) {
   event.target.playVideo();
-  startVideoSyncTimer();
+  window.synchronizer.startVideoSyncTimer();
```

Comment:
Why window here? Wht not pass this into the video.js class? If this is the plan please add a TODO.

Reply:
A TODO for this is located in `View.js` where `synchronizer` is created. Do I need to add a TODO everywhere I do `window.synchronizer`?

Reply:
Ah I see, as long as there is a TODO.